IdentityCore/src/requests/result/MSIDTokenResult.h

```diff
@@ -65,8 +71,8 @@ NS_ASSUME_NONNULL_BEGIN
 /* Token response from server */
 @property (nonatomic, nullable) MSIDTokenResponse *tokenResponse;
 
-/* Broker app version used for brokered authentication */
-@property (nonatomic, nullable) NSString *brokerAppVersion;
```

Comment:
We do a braking change for old clients here. Should we keep it and introduce metadata as a new property?